python/cudf/tests/test_string.py

```diff
@@ -324,6 +344,17 @@ def test_string_cat(ps_gs, others, sep, na_rep):
     expect = ps.str.cat(others=pd_others, sep=sep, na_rep=na_rep)
     if isinstance(others, pd.Series):
         others = Series(others)
+    elif isinstance(others, (list, tuple)):
+        new_others = []
+        for i in others:
+            if isinstance(i, pd.Series):
+                new_others.append(Series(i))
```

Comment:
Why do we need to explicitly convert the Pandas Series to a cudf Series here?

Reply:
We are explicitly converting pd Series to cudf Series as `others` is parameterized with pd.Series and for others we will need it to be a cudf.Series instead of pd.Series.

Reply:
Your logic above should pass it to the cudf.Series constructor which should handle the conversion of a pd.Series for us automagically

Reply:
Ah, Okay. Looks like previous implementation wasn't supporting such behavior. I will add that support too 👍

Reply:
@kkraus14 Added support for pd.Series too.